browser/registry.go

```diff
@@ -0,0 +1,57 @@
+package browser
+
+var cmdRegistry map[string](func(Command, chan *ResultSet))
```

Comment:
In fact I would probably extend the interface to:
```go
type CommandHandler func(context.Context, Command, chan<- *ResultSet)
```

Check out golang context package, it's how you signal cancellation... I'm mostly thinking that if in main we receive a signal like SIGTERM, we want to abort all ongoing operations and exit.